iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -498,6 +498,18 @@ def scheduling_parameters(self, value: Any):
             raise ValueError("Scheduling parameters must be a dict")
         self.fx_node.scheduling_parameters = value
 
+    @property
+    def expanded_dims(self) -> dict[IndexSymbol, int]:
```

Comment:
Can we add documentation that mentions that it is used to store it's "expansion id"/"dim_query"?

Reply:
added.